bouncer/blacklist/admin.py

```diff
@@ -2,7 +2,14 @@
 from blacklist import models
 
 
-@admin.register(models.IPEntry, models.EmailEntry, models.EmailHostEntry)
+@admin.register(models.IPEntry, models.EmailHostEntry)
 class EntryAdmin(admin.ModelAdmin):
-    fields = ["entry_value", "reason"]
-    search_fields = ["entry_value"]
+    fields = ["entry_value", "lower_case_entry_value", "reason"]
+    search_fields = ["lower_case_entry_value", "entry_value"]
+
+
+@admin.register(models.EmailEntry)
+class EmailEntryAdmin(admin.ModelAdmin):
+    fields = ["entry_value", "lower_case_entry_value", "hashed_value", "reason"]
+    search_fields = ["lower_case_entry_value", "entry_value"]
```

Comment:
There is no need for `lower_case_entry_value` to be searchable, `entry_value` will suffice. Also, it would be nice to be able to search against the `hashed_value` here, so that admins can also search against that.